xbmc/utils/BitstreamConverter.cpp

```diff
@@ -72,6 +72,34 @@ enum {
     HEVC_NAL_SEI_SUFFIX = 40
 };
 
+enum {
```

Comment:
right, but the fact that the old code does not comply to our coding guideline does not justify that new code does not have to do so either. what do you think?